federation/agent.buildkite.com/config.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 The Sigstore Authors
```

Comment:
nit: s/2022/2023 throughout for all new files

Reply:
Are you sure, it still feels like 2022 to me...

Reply:
It was 2022 when I wrote it 😅 but I'll update for merge